packaging/greenboot/microshift-running-check.sh

```diff
@@ -25,19 +25,21 @@ function forced_termination() {
     exit 1
 }
 
-# Check preconditions for existence of topolvm deployment.
+# Check preconditions for existence of lvms deployment.
 # Adapted from MicroShift code.
 #
 # args: None
-# return: 0 if topolvm readiness should be checked, 1 otherwise
-function topolvmShouldBeDeployed() {
+# return: 0 if lvms readiness should be checked, 1 otherwise
+function lvmsShouldBeDeployed() {
     if ! hash vgs 2>/dev/null; then
         return 1
     fi
-
     if [ -f /etc/microshift/lvmd.yaml ]; then
         return 0
     fi
+    if [ -f /etc/microshift/config.yaml ]; then
+        lvmsDriverShouldExist "/etc/microshift/config.yaml"
```

Comment:
`lvmsDriverShouldExist` returns 0/1, I don't see the result being used?
Also, it doesn't take any arguments, yet here we're passing path to the config

Reply:
Ah, 2AM coding :/ Will fix this.